test/evergreen.yml

```diff
@@ -1586,6 +1586,27 @@ tasks:
           display_name: Coverage report
           remote_file: wiredtiger/${build_variant}/${revision}/coverage_report/coverage_report_${build_id}.html
 
+  - name: test-wtperf
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: --enable-strict --enable-diagnostic
```

Comment:
I think it's better to separate the compile task but in this case we only have one test. It's up to you.

Reply:
I agree this should be separated, actually reusing the existing `compile` task is good enough (as . the configure options are a subset of that of `compile` task).